java/opendcs/src/main/java/ilex/util/PasswordFileEditor.java

```diff
@@ -396,26 +402,34 @@ protected boolean isOkToQuit( )
 	* @param echo
 	* @return the response
 	*/
-	private String getResponse( String prompt, boolean echo )
+private String getResponse(String prompt, boolean echo) 
```

Comment:
We should just move this to a static utility function. It's identical in the two files.